responsive-web-design/css-typography/index14.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Nutrition Label</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">
+  </head>
+  <style>
+    body {
+      font-family: 'Open Sans', sans-serif;
+      font-size: 16px;
+    }
+
+    .label {
+      border: 2px solid black;
+      width: 270px;
+      margin: 20px auto;
+      padding: 0 7px;
+    }
+  </style>
+  <body>
+    <div class="label">
+      <h1>Nutrition Facts</h1>
+      <p>8 servings per container</p>
+      <p>Serving size 2/3 cup (55g)</p>
+    </div>
+  </body>
+</html>
+
+<!-- 
+If you inspect the label using your browser's developer tools, you'll see that it's actually 288 pixels wide instead of 270.
+This is due to the CSS box model, which includes the dimensions of the padding and border when calulating each element's width and height.
+To fix this, add the following CSS rules to the top of your style block, above the body rule:
+```
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
```

Comment:
I'd like to see `*` introduced on its own as well as the `:before` and `:after` keywords. These are very important concepts in CSS and they shouldn't be glossed over. If you don't think its appropriate to introduce them in this challenge set then make a textual note that the user will learn more about them in a later challenge (or by visiting an external resource).

Reply:
Yes, that's a very good point. Ended up reworking this to introduce the `*` selector and used that to apply `box-sizing: border-box;` to every element. I thought it would be better to introduce pseudo selectors in a different CSS project.